docs/content/cache.md

```diff
@@ -182,6 +182,28 @@ and password) in your remote and it will be automatically enabled.
 Affected settings:
 - `cache-workers`: _Configured value_ during confirmed playback or _1_ all the other times
 
+##### Certificate Validation #####
+
+When the Plex server is configured to only accept secure connections, it is
+possible to use `.plex.direct` URL's to ensure certificate validation succeeds.
```

Comment:
Shouldn't this say required to use the *.plex.direct URL's for the certificate to evaluate at all without issue?

Reply:
Not necessarily, the Plex server can also be configured to use a user provided certificate in addition to the `.plex.direct` one. Another possibility is to put a reverse proxy in front of the Plex server with a custom certificate.

There is also the option to use #2555 once it's merged (which also needs to be documented). Maybe it's better to wait until it is done and then update this PR.